README.md

```diff
@@ -80,6 +80,18 @@ Additionally, there is a screencast available on YouTube, guiding you through th
 [![DiffDetective Demonstration](docs/yt_thumbnail.png)](https://www.youtube.com/watch?v=q6ight5EDQY)
 
 
+## Supported Differencing Algorithms
+
+In principle, any generic differencing algorithm (i.e, any algorithm that may operate on text or trees) can be made variability-aware with DiffDetective, as explained in our demo paper (see below). Some algorithms are integrated directly in the DiffDetective library, while others come as additional Maven projects.
+
+### Shipped with DiffDetective
+- Git Diff
```

Comment:
We use JGit as our Git implementation. [JGit only supports the Meyers and Histogram diff algorithms.](https://javadoc.io/static/org.eclipse.jgit/org.eclipse.jgit/7.2.0.202503040940-r/org.eclipse.jgit/org/eclipse/jgit/diff/DiffAlgorithm.SupportedAlgorithm.html) However, the original Git implementation supports more algorithms (see `man git diff | less -p '--diff-algorithm'`). Thus we should mention which Git implementation we use.
```suggestion
- Git Diff as implemented by [JGit](https://github.com/eclipse-jgit/jgit)
```